applications/Subcoin.md

```diff
@@ -0,0 +1,150 @@
+
+### Project Details
+
+**Key Deliverables**
+
+- crate `pallet-bitcoin`: This crate implements the core logic of Bitcoin protocol, handling the transaction processing and UTXO set management.
+- crate `runtime`: A lean Bitcoin specialized Substrate runtime integrating only two pallets frame-system and pallet-bitcoin.
+- crate `sc-consensus-nakamoto`: This crate offers an import queue for Bitcoin block import and full block verification.
+- crate `bitcoin-network`: This crate is responsible for basic Bitcoin P2P network functionalities, facilitating initial block download from other nodes.
+- crate `node`: This crate orchestrates the integration of all components mentioned above, providing a binary `subcoin` for running a fully functional Bitcoin node.
+
+**Non-Goals**
+
+This application does not aim to replicate the full feature set of a comprehensive Bitcoin client like **`bitcoind`**. As such, functionalities such as wallet management, mining, and mempool operations are outside the scope of this project, which are expected to be pursued in the future.
+
+### Ecosystem Fit
+
+The impact of this project extends beyond the Substrate and Polkadot ecosystems, contributing to the diversity of Bitcoin protocol implementation within the broader blockchain landscape.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Name of team leader: Liu-Cheng Xu
+
+### Contact
+
+- **Contact Name:** Liu-Cheng Xu
+- **Contact Email:** [xuliuchengxlc@gmail.com](mailto:xuliuchengxlc@gmail.com)
+- **Website:** https://github.com/liuchengxu
+
+### Legal Structure
+
+- Individual
+
+### Team's experience
+
+- Liu-Cheng Xu
+    
+    He is a seasoned protocol-level blockchain engineer with extensive experience with the Substrate framework. He was the core developer and subsequent tech lead of ChainX project from 2018 to 2021. Post-ChainX, he embarked on a venture with the permanent storage project Canyon and received the grant [canyon_network](https://github.com/w3f/Grants-Program/blob/df34fc617c88346411319b4293db5eb9cb2769b9/applications/canyon_network.md) for implementing the Proof Of Access consensus with the Substrate ecosystem. Due to regulatory changes in his country, Canyon was discontinued. He later transitioned to Subspace Labs as the lead protocol engineer, spearheading the execution layer development in Subspace's modular solution from 2021 to 2023.
+    
+
+### Team Code Repos
+
+- https://github.com/subcoin-project/subcoin
+- https://github.com/liuchengxu
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/liucheng-xu-2a639b116/
+
+## Development Status :open_book:
+
+A preliminary prototype of a Bitcoin node has been developed, featuring the capability to import Bitcoin blocks directly from the `bitcoind` database. A document [Design Notes](https://www.notion.so/915ff68a2b3c43b1ac37048c88d551ed?pvs=21) has been compiled to outline implementation considerations and provide guidance for ongoing development efforts
```

Comment:
Let me know if anyone would like to review this preliminary prototype, it's now in a private repo because the code is a mess and unpolished.